mautic/mautic
# [UI] Remove panel class from page lists

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This pull request addresses the issue of unwanted borders appearing in page lists due to the current styling of the body container. By removing the panel class from these lists, we can achieve a cleaner and more consistent appearance across the interface. 

This change not only resolves the visual inconsistency but also enhances the overall user experience by ensuring that the design remains sleek and modern. Users interacting with the page lists will benefit from a more polished look




| Before                                 | After
| -------------------------------------- | ---
| ![image](https://github.com/user-attachments/assets/7c0d5338-963a-4504-9f37-9649dc654cbf)                                        | ![image](https://github.com/user-attachments/assets/a4b2c20e-6601-4213-9fe0-06253fcc04b7)




---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the Contacts page, create a new contact, save it, and then close it and go back to the list to see the table.
